frontend/sozisel-app/src/components/SessionResultScreen/SessionResultRecording/SessionResultRecording.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactElement, useEffect } from "react";
+
+import { CircularProgress } from "@material-ui/core";
+import { SessionRecordingAnnotatedPlayer } from "./SessionRecordingAnnotatedPlayer/SessionRecordingAnnotatedPlayer";
+import SessionRecordingUpload from "./SessionRecordingUpload";
+import { useGetRecordingBySessionQuery } from "../../../graphql";
+
+interface SessionResultRecordingProps {
+  sessionId: string;
+}
+
+export function SessionResultRecording({
+  sessionId,
+}: SessionResultRecordingProps): ReactElement {
+  const { data, loading, error } = useGetRecordingBySessionQuery({
+    variables: { id: sessionId },
+  });
+
+  useEffect(() => {
+    if (error) {
+      console.error(error);
+    }
+  }, [error]);
+
+  if (loading) {
+    return <CircularProgress />;
+  }
+
+  if (error) {
```

Comment:
this can be a last statement so there is no need to check for error as the next if will not be loaded

Reply:
Not really, the last statement is upload recording component. Nevertheless, I think putting a cicrular progress bar on error is decent enough.